packages/react/src/primitives/shared/utils.ts

```diff
@@ -34,7 +34,7 @@ export const getConsecutiveIntArray = (
   return Array.from({ length }, (_, idx) => idx + start);
 };
 
-export type Modifiers = string | number | null;
+export type Modifiers = string | number | null | boolean;
```

Comment:
Should this allow `undefined` instead, so we can allow undefined params without having to wrap each one in Boolean?
```suggestion
export type Modifiers = string | number | null | undefined;
```

Reply:
hmm, I see you already added `?` below, but not sure why we should add `boolean`.